lib/aerospike/aerospike_exception.rb

```diff
@@ -39,7 +64,11 @@ def initialize(timeout, iterations, failed_nodes=nil, failed_connections=nil)
         @failed_nodes = failed_nodes
         @failed_connections = failed_connections
 
-        super(ResultCode::TIMEOUT)
+        super(ResultCode::TIMEOUT, "Timeout after #{iterations} attempts!")
+      end
+
+      def retryable?
```

Comment:
Do we have to overwrite `retryable?` here? We handle `ResultCode::TIMEOUT` in the super function:

https://github.com/castle/aerospike-client-ruby/pull/5/files#diff-fc56053424973707d0ca764bb54a373164030ab4ec934de00f76a39dbec6654fR42